source/layouts/layout.erb

```diff
@@ -22,10 +22,8 @@
 
     <!-- Google Search Console ownership verification -->
     <meta name="google-site-verification" content="niWnSqImOWz6mVQTYqNb5tFK8HaKSB4b3ED4Z9gtUQ0" />
-    <% 
-    # The code below is set to false so as not to enable Google Analytics for ICO compliance. Re-enable when opt-in as been done
-    # Original code is - if config.analytics != '' -
-    %>
+    <% # The code below is set to false so as not to enable Google Analytics for ICO compliance. Re-enable when opt-in as been done %>
+    <% # Original code is - if config.analytics != '' - %>
```

Comment:
I think this can be removed in favour of using git history.

Reply:
Sure